code/API_definitions/qod-api.yaml

```diff
@@ -273,7 +290,13 @@ paths:
       tags:
         - QoS Sessions
       summary: Delete a QoS session
-      description: Free resources related to QoS session
+      description: |
+        Free resources related to QoS session
```

Comment:
What about "Release resources" instead of "Free resources?" Since free can have several meanings, which could be ambiguous.

Reply:
I agree, "release" is the better choice also for me. I haven't introduced it with this PR, but instead used it from here also in two other occurrences. Replaced with "release" in all three occurrences now.

Reply:
Done.